src/Microsoft.AspNetCore.Mvc.Razor.Extensions/InjectDirectiveTargetExtension.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.AspNetCore.Mvc.Razor.Extensions
 {
     public class InjectDirectiveTargetExtension : IInjectDirectiveTargetExtension
     {
-        private const string RazorInjectAttribute = "[global::Microsoft.AspNetCore.Razor.Internal.RazorInjectAttribute]";
+        private const string RazorInjectAttribute = "[global::Microsoft.AspNetCore.Mvc.Razor.Internal.RazorInjectAttribute]";
```

Comment:
Yeah, I was using the Integration tests to make sure there weren't any other mis-named things but you're right, I still could/should have seperated.